lib/blocs/home_bloc/home_state.dart

```diff
@@ -3,9 +3,19 @@ part of 'home_bloc.dart';
 @immutable
 abstract class HomeState {}
 
-class LoadingState extends HomeState {}
-
-class AuthenticatedState extends HomeState {}
+class InitialState extends HomeState {}
+
+class AuthenticatedState extends HomeState {
+  final Future<List<Book>> popularStream;
```

Comment:
Incorrect name. Please remove word Stream. Can be names popularBooks. The same about releases

Reply:
done